.gitmodules

```diff
@@ -14,3 +14,6 @@
 	path = extern/ParseOggVorbis
 	url = https://github.com/albertz/ParseOggVorbis.git
 	branch = master
+[submodule "extern/warprnnt/warp-transducer"]
+	path = extern/warprnnt/warp-transducer
+	url = git@github.com:HawkAaron/warp-transducer
```

Comment:
I think the `git@github.com` url only works if you have an account at GitHub, and configured the GitHub SSH key. The `http` url always works, also without GitHub account. That's why I have used `http` for the others.

Reply:
Btw, because of this, all Travis tests fail. So we definitely need the `http` url here.

Reply:
Oh, I see. Changed that